src/modules/UI/Settings/selectors.js

```diff
@@ -37,13 +42,20 @@ export const getDenominations = (state: State, currencyCode: string) => {
 
 export const getDisplayDenominationKey = (state: State, currencyCode: string) => {
   const settings = getSettings(state)
-  const currencySettings = settings[currencyCode]
-  const selectedDenominationKey = currencySettings.denomination
+  let currencySettings, selectedDenominationKey
+  if (settings[currencyCode]) {
+    currencySettings = settings[currencyCode]
+    selectedDenominationKey = currencySettings.denomination
+  } else {
+    let tokenInfo = settings.customTokens.find((token) => token.currencyCode === currencyCode)
+    currencySettings = tokenInfo
+    selectedDenominationKey = currencySettings.multiplier
+  }
   return selectedDenominationKey
 }
 
 export const getDisplayDenominationFromSettings = (settings: any, currencyCode: string) => {
-  const currencySettings = settings[currencyCode] || isoFiatDenominations[currencyCode]
+  const currencySettings = settings[currencyCode] || isoFiatDenominations[currencyCode] || settings.customTokens[currencyCode]
```

Comment:
why are user added tokens different than all the other currency settings?

Reply:
Again, because a user could hypothetically enter a currency key (currencyCode) that matches one of the other settings in Redux. I realize using an array and doing multiple bitwise operations isn't the most elegant of solutions.